templates/nextjs/src/hooks/useActiveWallet.tsx

```diff
@@ -6,6 +6,9 @@ import { AppWallet } from "@/lib";
 /**
  * burner -> a burner wallet embedded inside of the template app and stored in local storage
  * browser -> a wallet connected via a browser extension like the Fuel Wallet
+ *
+ * Whenever a browser wallet is connected, this hook will return an instance of the browser wallet.
+ * Otherwise, it will return an instance of a local burner wallet.
```

Comment:
Do you think we should add this comment to the hook method?

https://github.com/FuelLabs/fuels-ts/pull/2772/files#diff-d04dd1651db7f4a288fbabab7c9a4f7cd141068f0ed89e5aea1ddc7ee75ce34bR62

Reply:
@petertonysmith94 mate that link seems to be broken. Can you check?

Reply:
Apologies - [here](https://github.com/FuelLabs/fuels-ts/blob/master/templates/nextjs/src/hooks/useActiveWallet.tsx#L59-L60).